.appveyor.yml

```diff
@@ -72,6 +72,9 @@ install:
   # For all other archs (including, say, arm), use the x64 python.
   - cmd: if %arch%==x86 (set MESON_PYTHON_PATH=C:\python35) else (set MESON_PYTHON_PATH=C:\python35-x64)
 
+  # Skip CI requires python
+  - cmd: python ./skip_ci.py --base-branch-env=APPVEYOR_REPO_BRANCH --is-pull-env=APPVEYOR_PULL_REQUEST_NUMBER
```

Comment:
There was a built in way to skip it on AppVeyor.

Reply:
You mean [commit-files](https://www.appveyor.com/docs/how-to/filtering-commits/#commit-files-github-and-bitbucket-only) I guess, the downside is that it would skip build for direct pushes as well. Also since there is no counterpart in Travis, we may end up with inconsistent behavior. One of the advantage of having script is that it prints error message explaining why build is skipped.